src/sessionization.py

```diff
@@ -20,90 +20,80 @@ def time_diff(dt, dt0):
     diff = int(diff_obj.total_seconds())
     return diff
 
-def clean_active_log(num):
+def clean_active_log(log_file, ip_dict, f2):
```

Comment:
- be careful with your naming of filehandles ```f2```  can be ```out_fh```
- same goes for ```f1``` which can be ```in_fh```
- sorry if I got them switched